.travis/install.sh

```diff
@@ -1,16 +1,33 @@
 #!/bin/bash
 
+cd "$(dirname "$(readlink -f "$0")")"
 set -ev
-
-cd source
 #install core of utplsql
-"$SQLCLI" $UT3_OWNER/$UT3_OWNER_PASSWORD@//$CONNECTION_STR @install.sql $UT3_OWNER
+"$SQLCLI" sys/$ORACLE_PWD@//$CONNECTION_STR AS SYSDBA <<-SQL
+pwd
+set feedback off
+set verify off
 
+@../source/create_utplsql_owner.sql $UT3_OWNER $UT3_OWNER_PASSWORD $UT3_TABLESPACE
+--only needed to run unit tests for utplsql v3, not required to run utplsql v3 itself
+grant select any dictionary to $UT3_OWNER;
```

Comment:
It is needed for ut_coverage to work.
That should be fixed/addressed separately.
I just noticed that yesterday.
Also the coverage should use dba_source for best results.